plugins/cliconf/vyos.py

```diff
@@ -83,9 +83,9 @@ def get_device_info(self):
             if match:
                 device_info["network_os_version"] = match.group(1)
 
-            match = re.search(r"HW model:\s*(\S+)", data)
+            match = re.search(r"(HW|Hardware) model:\s*(\S+)", data)
```

Comment:
```suggestion
            match = re.search(r"(?:HW|Hardware) model:\s*(\S+)", data)
```
If we're not using this information, there's no reason to capture it.